src/semgrep_agent/utils.py

```diff
@@ -28,10 +28,10 @@ def __init__(self, message: str) -> None:
 
 def debug_echo(text: str) -> None:
     """Print debug messages with context-specific debug formatting."""
-    if os.getenv("GITHUB_ACTIONS"):
-        prefix = "::debug::"
-    elif os.getenv("SEMGREP_AGENT_DEBUG"):
+    if os.getenv("SEMGREP_AGENT_DEBUG"):
```

Comment:
Switched these two so if SEMGREP_AGENT_DEBUG is set to 1 in env variiable no need to also set the Github actions debug flag in secrets. Or is there a reason we want to put debug flag behind secrets permissioning @underyx ?

Reply:
I think GHA does some funky parsing